### ISSUE TITLE
Change weights in loss function

### ISSUE THREAD

**Issue by Gasp34** [AlexeyAB/darknet#4106]
Hello

I just read this in https://doi.org/10.1111/2041-210X.13165

> Finally, we modified the training loss function to suppress false positives and account for the large amounts of empty space in the training images. We achieved this by increasing the weighting (from 0.5 to 2) given to the no-­object component of the multipart loss function described in Redmon et al. (2016)

They use a Keras implemenation of YOLOv3 to do it, is it possible to do it with Darknet ? 

Do you think it's a good idea ? 

Thanks !

**Comment by AlexeyAB** [comment]
Redmon et al. (2016) - it is about Yolo v2 rather than Yolo v3.

> We achieved this by increasing the weighting (from 0.5 to 2) given to the no-­object component of the multipart loss function

Do they write about it?
* https://github.com/AlexeyAB/darknet/blob/53160fa6662ea7e8c4095588db56cac7b339b917/src/region_layer.c#L271
* https://github.com/AlexeyAB/darknet/blob/53160fa6662ea7e8c4095588db56cac7b339b917/cfg/yolov2.cfg#L252

But there is initially `1` (not `0.5`)

----

There is already introduced parameter `iou_normalizer` in GIoU for Yolo v3: https://github.com/AlexeyAB/darknet/issues/3249
* https://github.com/AlexeyAB/darknet/blob/53160fa6662ea7e8c4095588db56cac7b339b917/src/yolo_layer.c#L293
* https://github.com/AlexeyAB/darknet/blob/53160fa6662ea7e8c4095588db56cac7b339b917/cfg/yolov3.coco-giou-12.cfg#L805
